src/njoy21/input/ACER/Card12.hpp

```diff
@@ -0,0 +1,17 @@
+class Card12 {
```

Comment:
The reason Card11 is repeated for the iopt values is because it's the exact same card. There is no Card12